src/frontend-samples/reality-data-sample/sampleSpec.ts

```diff
@@ -12,12 +12,13 @@ export function getRealityDataSpec(): SampleSpec {
     label: "Reality Data",
     image: "reality-data-thumbnail.png",
     iTwinViewerReady: true,
-    readme: async () => import("!!raw-loader!./readme.md"),
+    readme: async () => import("!raw-loader!./readme.md"),
+    walkthrough: async () => import("!annotation-loader!./annotations.md"),
     files: () => [
-      { name: "RealityDataApi.ts", import: import("!!raw-loader!./RealityDataApi") },
-      { name: "RealityDataApp.tsx", import: import("!!raw-loader!./RealityDataApp"), entry: true },
-      { name: "RealityDataWidget.tsx", import: import("!!raw-loader!./RealityDataWidget") },
-      { name: "RealityData.scss", import: import("!!raw-loader!./RealityData.scss") },
+      { name: "RealityDataApi.ts", import: import("!annotation-raw-loader!./RealityDataApi") },
```

Comment:
The "annotation-raw-loader" will strip the annotation comments from the file and load the file the raw-loader does.